packages/gatsby-theme-store/src/sdk/searchBar/controller/index.ts

```diff
@@ -65,10 +65,10 @@ export const search = async (term: string) => {
   } else {
```

Comment:
I'm not really sure about this implementation. I'd rather keep the params but remove any map params in the first if